docs/start/configuration/index.rst

```diff
@@ -70,7 +68,7 @@ You can override that with something like:
         options:
           default_broadcast: 192.168.1.255
 
-Or create your own:
+The target can also be renamed:
```

Comment:
it's more another target rather than a renamed target. By adding your own, the default `lan` target still exists